hack/setup-rbac/main.go

```diff
@@ -2,71 +2,25 @@ package main
 
 import (
 	"context"
-	"fmt"
 	"os"
-	"time"
 
 	"github.com/Azure/azure-sdk-for-go/services/authorization/mgmt/2015-07-01/authorization"
 	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/sirupsen/logrus"
 
 	"github.com/openshift/openshift-azure/pkg/fakerp"
 	"github.com/openshift/openshift-azure/pkg/util/azureclient"
 )
 
-// listActions lists all the mutating actions carried out on a resource group in
-// the last 6 hours, along with the principal that carried out the action
-func listActions(ctx context.Context, resourceGroupName string) error {
```

Comment:
@jim-minter was this intentional removal?

Reply:
@mjudeikis seems this was not being used anywhere

Reply:
yeah, it was annoying me since the insights change.  I don't think we're going to use that code again.